core/server/master/src/main/java/alluxio/master/file/meta/InodeDirectory.java

```diff
@@ -51,6 +51,8 @@ public Object getFieldValue(Inode<?> o) {
 
   private boolean mDirectChildrenLoaded;
 
+  private AccessControlList mDefaultAcl;
```

Comment:
Is this supposed to be `DefaultAccessControlList` or `AccessControlList`?

Also, is there a notion of an "empty" acl, instead of a `null` acl? `null` signifying empty could result in bugs.

Reply:
documented as discussed.